Tests/VB.Specs/VB.Specs.vbproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>VB.Specs</RootNamespace>
+    <LangVersion>16.9</LangVersion>
```

Comment:
VB doesn't have a version 13.0.
In main it works by coincidence because VB does have a version 12.0.